homeassistant/components/tellduslive/__init__.py

```diff
@@ -210,7 +206,7 @@ class TelldusLiveClient:
 
     def __init__(self, hass, config, session):
         """Initialize the Tellus data object."""
-        self.entities = []
+        self._known_devices = set()
```

Comment:
I'm a bit puzzled what's the best datatype for this, `[]` is fast to insert and `set()` is significantly faster for lookups. Since I expect that we'll do more [lookups](https://github.com/home-assistant/home-assistant/pull/18780/files#diff-6182200c1051279a736e0dd1a80942d9R254) than [finding new devices](https://github.com/home-assistant/home-assistant/pull/18780/files#diff-6182200c1051279a736e0dd1a80942d9R263) I went with a `set()`.